docs/src/main/resources/microsite/sass/_variables.scss

```diff
@@ -0,0 +1,55 @@
+// Variables
+// -----------------------------------------------
+// -----------------------------------------------
+// Google Fonts
+// -----------------------------------------------
+@import url('https://fonts.googleapis.com/css?family=Cairo:400,600,700&display=swap');
+
+// Colors
+// -----------------------------------------------
+$main-color: #3c59fe;
+$masthead-background: #f3f5ff;
+$text-color: #1f1c49;
+$footer-text-color: rgba(255,255,255,0.5);
+$white: #FFF;
+$sidebar-background: #d5dbff;
+$sidebar-active-color: #ccd3ff;
+$sidebar-head-active-color: lighten($main-color, 4%);
+$gray-primary: rgb(31, 28, 73);
+
+// Typography
+// -----------------------------------------------
+$base-font-family: 'Cairo', sans-serif;
+$header-font-family: $base-font-family;
+$code-font-family: 'Source Code Pro', monospace;
```

Comment:
I guess this is coming from `sbt-microsites`?  Otherwise is lacking from the 3rd party fetching.  And in case is coming from `sbt-microsites`, it might not be needed here, is it?

Reply:
Ok it is removed now and is using sbt-microsites code-font-family